src/emulator/renderer/src/texture_palette.cpp

```diff
@@ -16,7 +16,7 @@ void palette_texture_to_rgba_4(uint32_t *dst, const uint8_t *src, size_t width,
     for (size_t y = 0; y < height; ++y) {
         uint32_t *const dst_row = &dst[y * width];
         const uint8_t *const src_row = &src[y * stride];
-        for (size_t x = 0; x < width; ++x) {
+        for (size_t x = 0; x < width - 1; ++x) {
```

Comment:
Maybe `++x` should be `x += 2` here. What do you think? Then we can revert the `- 1`.

Reply:
I'll put the x +=2 but leave the condition

Reply:
It shouldn't really make a difference, as I think the widths are always even.